meta-lxatac-bsp/recipes-kernel/linux/files/defconfig

```diff
@@ -13,9 +13,10 @@ CONFIG_LD_VERSION=24100
 CONFIG_LLD_VERSION=0
 CONFIG_CC_HAS_ASM_GOTO_OUTPUT=y
 CONFIG_CC_HAS_ASM_GOTO_TIED_OUTPUT=y
+CONFIG_GCC_ASM_GOTO_OUTPUT_WORKAROUND=y
 CONFIG_CC_HAS_ASM_INLINE=y
 CONFIG_CC_HAS_NO_PROFILE_FN_ATTR=y
-CONFIG_PAHOLE_VERSION=0
+CONFIG_PAHOLE_VERSION=124
```

Comment:
Can you check that this is not the host system's `pahole(1)`? e.g. with `oe-run-native`.

Reply:
Oh, I'll check that. I would have hoped that that would trigger some kind of host leakage detection, but I've been disappointed before on that front.

Reply:
Thanks for investigating. IMO, the OE-core commit should have kept the behavior introduced with https://lore.kernel.org/all/20220324162351.2759862-1-a.fatoum@pengutronix.de/ for the case that `KERNEL_DEBUG` is missing. Your workaround here looks good.